provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/controller/practitioner/PractitionerTransformer.java

```diff
@@ -111,34 +122,43 @@ public Practitioner apply(PractitionerWrapper ppmsData) {
                 .build());
   }
 
-  ContactPoint telecom(String system, String value) {
-    if (value == null) {
+  List<ContactPoint> providerContactsTelecom(ProviderContactsResponse.Value source) {
+    if (source == null || allBlank(source.mobilePhone())) {
       return null;
     }
-    return ContactPoint.builder()
-        .system(EnumSearcher.of(ContactPoint.ContactPointSystem.class).find(system))
-        .value(value)
-        .build();
+    List<ContactPoint> telecoms = new ArrayList<>();
+    if (source.mobilePhone() != null) {
+      telecoms.add(telecom("phone", source.mobilePhone()));
+    }
+    return telecoms;
```

Comment:
There should be a check to return `null` if `telecoms` is empty. The transformers in data-query use this method:
https://github.com/department-of-veterans-affairs/health-apis-data-query/blob/master/data-query/src/main/java/gov/va/api/health/dataquery/service/controller/Transformers.java#L187